node-process-argv/echo.js

```diff
@@ -0,0 +1 @@
+console.log(process.argv[process.argv.length - 1]);
```

Comment:
This isn't correct. If you were to execute
`node echo 123 abc`
this would log
`abc`